### DISCUSSION TITLE
Starting mkdocs with pm2

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5266] [Q&A] [answered]
Hello, currently I launch my mkdocs with `screen -S docs mkdocs serve -a 127.0.0.1:4000` - would it be possible to start it with pm2?

**Comment by kamilkrzyskow** [comment]
After a quick search I get that pm2 was designed with Node applications in mind.

So I searched if it can run Python scripts too, and it seems that it does.

https://stackoverflow.com/a/34189393

`pm2 start mkdocs --interpreter=python`

However, I'm not sure if you can pass arguments in the command.
There are some examples of some `ecosystem.config.json` files that seem to allow to pass arguments.

**Comment by squidfunk** [comment, marked as answer]
Please note that the MkDocs development server [is not meant for hosting](https://github.com/mkdocs/mkdocs/issues/2108#issuecomment-627533212), only for previewing your site. It is inefficient and insecure. You should build your site with `mkdocs build` and then use __any web server__, e.g. nginx, for hosting your content.
